### ISSUE TITLE
[question] 0 (zero) bytes items in JFrog Artifactory after conan upload

### ISSUE THREAD

**Issue by SzBosch** [conan-io/conan#14536]
### What is your question?

Hello together,

since 17.8. we are facing very often the items of a conan package in JFrog Arifactory to have 0 (zero) bytes after conan upload command:

- conan_package.tgz
- conaninfo.txt
- conanmanifest.txt

Sometimes only one of them is corrupted, sometimes 2 and sometimes all 3 of them.

We cannot see any difference in the conan client console output compared to earlier. There is no error.
As a consequence such packages can not be installed. Recreating and uploading them again does not solve the problem.

Environment: conan 1.60.1 and JFrog Cloud.

There was a maintenance window on jfrog.io right before we observed the problem the first time.

Can it be that there was an incompatibility with conan introduced by this maintenance window on jfrog.io?
Are there other users with the same issue out there?

### Have you read the CONTRIBUTING guide?

- [X] I've read the CONTRIBUTING guide

**Comment by memsharded** [comment]
Hi @SzBosch 

Thanks for your report. It would be good to report the current JFrog Artifactory version if possible, and if you know in which version it was before the 17-August update, that would help too.

So far we are not aware of any other reports similar to this one. If you are a JFrog customer it makes sense to report this to JFrog support too.

**Comment by SzBosch** [comment]
Hi @memsharded, there is no version number displayed, it is just the "JFrog Cloud" version.

In the `artifactory-service.log` I see a lot messages matching this regex:

`Deploy to.+(conanfile.py|conanmanifest.txt|conaninfo.txt|conan.+\.tgz).+Content-Length: 0`

Are "Content-Length: 0" entries just normal?

We are cross-checking in parallel with JFrog support but we are happy for every input which can help.

Is there anything we can activate or instrument in conan to display how much bytes of the individual elements are actually uploaded?

**Comment by drussel** [comment]
We are seeing this at the moment too. Conan 1.60.2. Artifactory also. In addition to empty files, I think we have also had cases where the files have been swapped between packages (potentially packages that were uploaded around the same time). But mostly have been focussed on getting things unblocked.

Force pushing one at a time from a good local seems to sometimes fix the repository. But not always.

My working hypothesis had been that we are uploading in parallel and somehow the uploading Conans were stepping on one another (eg using the same temp files), but don't have a strong basis for this as it is intermittent. But could also be artifactory tripping up with too many incoming changes?

**Comment by drussel** [comment]
In case it's useful I think the message below is caused by non-empty garbage files (it would be nice if the error message would include which, as it is from our CI system). 
```
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/client/command.py", line 2216, in run
[2023-08-22T02:49:24.683Z]     method(args[0][1:])
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/client/command.py", line 814, in info
[2023-08-22T02:49:24.683Z]     self._outputer.json_info(deps_graph, json_arg, os.getcwd(), show_paths=args.paths)
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/client/conan_command_output.py", line 258, in json_info
[2023-08-22T02:49:24.683Z]     data = self._grab_info_data(deps_graph, grab_paths=show_paths)
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/Traceback (most recent call last):
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/tools/conan/run-conan.py", line 125, in <module>
[2023-08-22T02:49:24.683Z] python3.10/site-packages/conans/client/conan_command_output.py", line 206, in _grab_info_data
[2023-08-22T02:49:24.683Z]     node_times = self._read_dates(deps_graph)
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/client/conan_command_output.py", line 91, in _read_dates
[2023-08-22T02:49:24.683Z]     manifest = self._cache.package_layout(ref).recipe_manifest()
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/paths/package_layouts/package_cache_layout.py", line 187, in recipe_manifest
[2023-08-22T02:49:24.683Z]     return FileTreeManifest.load(self.export())
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/model/manifest.py", line 92, in load
[2023-08-22T02:49:24.683Z]     return FileTreeManifest.loads(text)
[2023-08-22T02:49:24.683Z]   File "/Users/swp_jenkins_mac/JenkinsBuild/FlowCI/build/venv/lib/python3.10/site-packages/conans/model/manifest.py", line 78, in loads
[2023-08-22T02:49:24.683Z]     the_time = int(tokens[0])
[2023-08-22T02:49:24.683Z] ValueError: invalid literal for int() with base 10: '\x1f\x08\x08\x00\x00\x00\x00\x02conan_sources.tgz\x00K0\x14\x07W^J[:\x18\x9c!M2I\x0f6{'
```

**Comment by SzBosch** [comment]
J.f.y.i.: On conan client side we can see 2 different error messages when executing a "conan install" on a package which is corrupted (with 0 bytes elements)  in Artifactory:

1. ERROR: Exception: <class 'conans.errors.ConanException'> Error while extracting downloaded file '.../conan_package.tgz' to ...
**file could not be opened successfully**

2. ERROR: Error retrieving manifest file for package '...' from remote (https://....jfrog.io/artifactory/api/conan/...): **'invalid literal for int() with base 10: '''**. [Remote: ...]


@drussel: Thank You for reporting.
Are you using also JFrog Cloud?
Since when do you have the same problem?

**Comment by SzBosch** [comment]
@memsharded: The JFrog Artifactory version in http response is displayed as:

X-JFrog-Version: Artifactory/7.67.0 76700900

Do you test from conan side with such setup:
https://jfrog.com/help/r/how-to-configure-https-with-nginx-for-artifactory-7-x/how-to-configure-https-with-nginx-for-artifactory-7.x
?

**Comment by memsharded** [comment]
Quick question @drussel

- Do you work at the same place as @SzBosch?
- Can you confirm your Artifactory, if it is a Cloud instance, and also check the version (in the headers)?

Yes, the error @SzBosch is reporting are expected if the files in the server side are corrupt and only 0 bytes


Some further questions for both:
- What % of uploads are failing? It is every one, often, or just very few?
- Is there any pattern of packages failing, like very big ones?
- Are the packages failing for any particular platform always, like always the same Windows binaries?
- Are the recipes upload failing also?
- Do you have enabled ``parallel`` uploads? (Try disabling if yes)

Any other hint that helps us reproduce would be great.

**Comment by drussel** [comment]
- I don't believe @SzBosch  and I work at the same place (Velo3D in my case). 
- Artifactory/7.67.2

- only a few percent of uploads failing
- seems to be small packages that fail (disproportionately header only ones), but statics aren't good
- I haven't seen anything consistently reproducible
- recipes get corrupted too, but, I think it may only occur when uploading artifacts, but not 100% sure (I haven never seen an issue when after deleting a package I only upload the one recipe, but the sample is small)
- we definitely enable parallel uploads, but in general are only uploading one package at a time. Will disable it. We were also running more than one conan upload process at the same time. We have disabled that, but it takes a bit for everything to propagate, so hard to say whether that fixed things.

I first explicitly noticed this over the weekend, but that was the first time I updated packages in a while, so the first time we were uploading in a bit.

**Comment by SzBosch** [comment]
-  What % of uploads are failing? It is every one, often, or just very few? -> I have no % figure at the moment, but it occurs quite often; let me update this post here later
-  Is there any pattern of packages failing, like very big ones? -> no pattern
-  Are the packages failing for any particular platform always, like always the same Windows binaries? -> we use only Linux here
-  Are the recipes upload failing also? -> yes, files like conanfile.py and conan_export.tgz are also affected
-  Do you have enabled parallel uploads? (Try disabling if yes) -> yes, we use "conan upload" with parameters "--parallel --retry 5 --retry-wait 5"; we were using this already before the issue came up

**Comment by memsharded** [comment]
Do you have revisions enabled, always, in all clients?

**Comment by SzBosch** [comment]
Yes, revisions are enabled.

**Comment by drussel** [comment]
We don't have revisions enabled on any as all sorts of stuff broken when we tried (sorry, fuzzy on details, just remember not having time to follow up with it).

**Comment by memsharded** [comment]
Thanks for the feedback.

@drussel, this would be important. Conan 2.0 uses revisions always, it is a critical feature for reproducibility, lockfiles, traceability, computation of the ``package_ids``, etc. I'd strongly recommend to activate it as soon as possible, and let us know about the things that might not be working for your case, and we can try to help to move forward.

**Comment by drussel** [comment]
Probably not useful, but one of our CI machines just had a package where the recipe was swapped with another one and I don't think the server state was ever corrupted for that recipe (and the recipe that was plopped in there was from a different artifactory server or conan center). Suggesting that downloads also could get tripped up. But all one off stuff and still no good pattern. But we do also do a lot of explicit downloading of conan packages and recipes in parallel to ensure we get binaries and such, but this packages was not one that we do that for.

Revisions is something I revisit periodically as part of moving towards conan 2 (which was blocked last I tried by not being able to cleanly drive conan from within cmake). We had to do a bunch of stuff to get reproducible builds prior to revisions and other tools being available and so have been only slowly unwinding those things and moving towards more standard mechanisms.

**Comment by memsharded** [comment]
> Revisions is something I revisit periodically as part of moving towards conan 2 (which was blocked last I tried by not being able to cleanly drive conan from within cmake).

If you are talking about the cmake-conan integration, please check the "develop2" branch in the repo, for Conan 2.0 that achieves completely transparent integration with CMake calling ``conan install`` thanks to the new CMake dependency providers: https://github.com/conan-io/cmake-conan/tree/develop2

**Comment by memsharded** [comment]
I have been trying today with some automated scripts that create and upload dozens of different packages to a 7.67 Artifactory instance in the cloud, but couldn't reproduce the issue so far.

**Comment by memsharded** [comment]
Can you please report in which cloud are you running your Artifactory? Cloud provider and region, if possible? Maybe it is something platform/network specific to the specific environment and not only the Artifactory cloud version?

**Comment by drussel** [comment]
We are AWS of some sort. Not sure how to tell the region?

So I think it might have to do with running multiple upload processes in parallel on a client (and potentially some data getting mixed up between them). Eg with the script below with a pool size of 1, I haven't seen issues, but when I increase the pool to 10, I do.

```
import sys
import subprocess
from pathlib import Path
from multiprocessing import Pool


def _get_remote(n):
    if n.endswith("velo3d/thirdparty"):
        return "external"
    elif n.endswith("velo3d/stable"):
        return "internal"
    else:
        return "conancenter-cache"


def _upload(pkg):
    conan = Path(sys.executable).parent/"conan"
    n, p = pkg
    r = _get_remote(n)
    if r != "conancenter-cache":
        return
    # force upload recipe
    cmd = [str(conan), "upload", f"{n}",
           "--parallel", "-c", "--force", "-r", r]
    print(f"calling {' '.join(cmd)}")
    subprocess.call(cmd)

    # update binary if appropriate
    cmd = [str(conan), "upload", f"{n}:{p}",
           "--parallel", "-c", "-r", r]
    print(f"calling {' '.join(cmd)}")
    subprocess.call(cmd)


if __name__ == '__main__':
    p = Path("conaninfo.txt").read_text().split("\n")
    packages = []
    seen = False
    for l in p:
        if l == "[full_requires]":
            seen = True
            continue
        if seen and not l:
            break
        if seen:
            packages.append(l.strip().split(":"))
            print(packages[-1])

    for n, p in packages:
        print(f"{n}: {_get_remote(n)}")

    p = Pool(10)
    p.map(_upload, packages)
```
This might be a "don't do that" situation (and I'm sure some of the bits of the script will make you cringe :-) But it makes upload way faster, particularly when having to generate the packages locally and being a logically const operation re the local conan install, seems like something that one would expect to work (given the locking mechanism in place).

@SzBosch would that fix with your experience?

**Comment by SzBosch** [comment]
> Can you please report in which cloud are you running your Artifactory? Cloud provider and region, if possible? Maybe it is something platform/network specific to the specific environment and not only the Artifactory cloud version?

Western Europe

**Comment by SzBosch** [comment]
> We are AWS of some sort. Not sure how to tell the region?
> 
> So I think it might have to do with running multiple upload processes in parallel on a client (and potentially some data getting mixed up between them). Eg with the script below with a pool size of 1, I haven't seen issues, but when I increase the pool to 10, I do.
> 
> ```
> import sys
> import subprocess
> from pathlib import Path
> from multiprocessing import Pool
> 
> 
> def _get_remote(n):
>     if n.endswith("velo3d/thirdparty"):
>         return "external"
>     elif n.endswith("velo3d/stable"):
>         return "internal"
>     else:
>         return "conancenter-cache"
> 
> 
> def _upload(pkg):
>     conan = Path(sys.executable).parent/"conan"
>     n, p = pkg
>     r = _get_remote(n)
>     if r != "conancenter-cache":
>         return
>     # force upload recipe
>     cmd = [str(conan), "upload", f"{n}",
>            "--parallel", "-c", "--force", "-r", r]
>     print(f"calling {' '.join(cmd)}")
>     subprocess.call(cmd)
> 
>     # update binary if appropriate
>     cmd = [str(conan), "upload", f"{n}:{p}",
>            "--parallel", "-c", "-r", r]
>     print(f"calling {' '.join(cmd)}")
>     subprocess.call(cmd)
> 
> 
> if __name__ == '__main__':
>     p = Path("conaninfo.txt").read_text().split("\n")
>     packages = []
>     seen = False
>     for l in p:
>         if l == "[full_requires]":
>             seen = True
>             continue
>         if seen and not l:
>             break
>         if seen:
>             packages.append(l.strip().split(":"))
>             print(packages[-1])
> 
>     for n, p in packages:
>         print(f"{n}: {_get_remote(n)}")
> 
>     p = Pool(10)
>     p.map(_upload, packages)
> ```
> 
> This might be a "don't do that" situation (and I'm sure some of the bits of the script will make you cringe :-) But it makes upload way faster, particularly when having to generate the packages locally and being a logically const operation re the local conan install, seems like something that one would expect to work (given the locking mechanism in place).
> 
> @SzBosch would that fix with your experience?

We are running only **one** conan command on the same agent/VM at the same time.
We are using the "--parallel" parameter on conan upload since a long time with no problems.

**Comment by thenextman** [comment]
We are also hitting this exact issue with jfrog/Artifactory. It was fine as of August 15th but broken since at least yesterday. We're pushing packages from GitHub Actions (in North America, if it's relevant). 

What % of uploads are failing? It is every one, often, or just very few? -> About 25-30% of packages, it seems like the same ones every time.

Is there any pattern of packages failing, like very big ones? -> Large and small

Are the packages failing for any particular platform always, like always the same Windows binaries? -> Failing on most (all?) the platforms we build (Windows/Linux/macOS/iOS/Android)

Are the recipes upload failing also? -> Seems like it

Do you have enabled parallel uploads? (Try disabling if yes) -> Yes we were using parallel uploads and also a matrix in GitHub to divide the work by platform. I've disabled all that and still hitting the issue.

cc: @awakecoding

**Comment by memsharded** [comment]
Thanks for the feedback @thenextman, just in case, but to be double sure, you are using Cloud JFrog instance, and could you please check the version of Artifactory is also 7.67?

So far, we haven't been able to reproduce it, still investigating :(

**Comment by memsharded** [comment]
Please @thenextman make also sure to file a support ticket to JFrog for this, with critical priority, so they also track it on their end (it might be a cloud production issue)

**Comment by thenextman** [comment]
@memsharded We have 7.67.3 here. I'm asking my ops team to make a ticket with jfrog.

**Comment by memsharded** [comment]
7.67 in the JFrog Cloud or on-prem @thenextman? Just in case, because after failing to find something in the code that could be causing the issue, I keep suspecting it could be some network/deployment issue of the cloud.

**Comment by thenextman** [comment]
> 7.67 in the JFrog Cloud or on-prem @thenextman? Just in case, because after failing to find something in the code that could be causing the issue, I keep suspecting it could be some network/deployment issue of the cloud.

Cloud

For the record, I'm currently using `--force` to push my packages (with no parallelization) and it seems to be ok, although it remains to be seen if the packages are actually "good" once pushed

**Comment by memsharded** [comment]
Team keep investigating, this is a challenging one.

Another extra feedback point, just in case: What Python version are you running in the failed cases in the Conan client side?

**Comment by thenextman** [comment]
@memsharded We did open a ticket with jfrog for this. I'll update if we get some news.

I had suspected an issue with the python version, because the error corresponded with an update to the GitHub runner image that we use (ubuntu-20.04) and it _did_ get a new python version (3.10). However, even pinning python back to 3.9 didn't help.

For now I was able to push all my packages in serial with `--force` and it worked (it was slow, but it worked)

**Comment by memsharded** [comment]
Update: Team keep working on this, still challenging, but it seems we have identified some interesting logs are we are narrowing the search, now working on a patch trying to get extended logs to keep narrowing. Thanks all for your patience.

**Comment by memsharded** [comment]
Extra piece of feedback just in case: It seems that all of you are affected, using Conan 1.60 (or close to it), but no one with Conan 2.0. Are you also uploading Conan 2.0 packages, in a parallel pipeline or something like that?

**Comment by AldBosch** [comment]
Me and @SzBosch work together. We don't upload Conan 2.0 packages at all.

**Comment by SzBosch** [comment]
Hello together,
in the meanwhile there are some news:

- We created a script to reproduce the issue.
- JFrog support is able to reproduce the issue on our instance (they were not able to reproduce it on an other (likely newly created) instance).
- We found an interesting missing entry the conan trace log.

Here is the pseudo-code of the script for reproduction (including a conan workflow):
```
for loop 1 to N:
  conan install ...
  conan build ...
  conan upload ... --parallel --check ... --force # note: the parameters --parallel --check do not make any difference here
  check consitency of files in Artifactory via curl & AQL query of the above uploaded package (see below)
  if any file has zero bytes (parsing the response.json from curl):
    give error message
    exit with non-zero
```
Note: There is simple dummy conanfile.py which does nothing but has just a build() function which writes a timestamp to a txt file to ensure the package payload always created a new PREV in each loop.

The script takes usually around 10 to 100 loops until the issue appears.

Here is the curl AQL query:
```
CONF_OUTPUT_FILE_OPTION="-o response.json"

AQL_QUERY="items.find({\"repo\": \"${CONF_ARTIFACTORY_REPO}\", \
            \"path\": {\"\$match\": \"${CONF_CONAN_USER}/${CONF_CONAN_NAME}/${CONF_CONAN_VERION}/${CONF_CONAN_CHANNEL}/*/package/${CONAN_PACKAGE_ID}/${CONAN_PACKAGE_REVISION}\"}, \
            \"type\": \"file\" \
            }).sort({\"\$desc\" : [\"created\"]})"
echo "AQL_QUERY=${AQL_QUERY}"

curl -u"${ARTIFACTORY_USER}":"${ARTIFACTORY_TOKEN}" -X POST -k -H 'Content-Type:text/plain' -i https://"${CONF_ARTIFACTORY_HOST}"/artifactory/api/search/aql ${CONF_OUTPUT_FILE_OPTION} \
        --data "${AQL_QUERY}"
```

When we run the script with
`export CONAN_TRACE_FILE=$(pwd)/conan_trace.log`
and then compare the trace of one loop which was ok with a loop in which a corrupted zero byte file was detected, we see actually missing entries, like:

package OK:
```
{"_action": "UPLOADED_RECIPE", "_id": "
{"_action": "ZIP", "dst": "/home/user/.
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "UPLOADED_PACKAGE", "_id": 
```

package CORRUPTED:
```
{"_action": "UPLOADED_RECIPE", "_id": "
{"_action": "ZIP", "dst": "/home/user/.
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
       <--- ENTRY MISSING !!! --->
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "REST_API_CALL", "duration"
{"_action": "UPLOADED_PACKAGE", "_id": 
```
Exactly the file(s) of the missing trace line(s) is/are the one(s) with zero bytes in Artifactory.

Do the trace entries "REST_API_CALL" appear only when the Artifactory Server confirms or also the the request is just sent?
I guess only then the server confirms, because there is a "duration" calculated.

Why does conan not abort here with an error message when such rest api call is "missing"? It just continues and exit with zero.

**Comment by memsharded** [comment]
Thanks for following up. This is the lead that we are following, apparently the server is responding as if the file was already there, and then missing the extra upload call, this is what the new patch is trying to understand better.

**Comment by SzBosch** [comment]
One more hint:

In the Artifactory WebUi when displaying the page of a 0 byte file, at the bottom where the checksums are, there is the following message:

_Client did not publish a checksum value. If you trust the uploaded artifact you can accept the actual checksum by clicking the 'Fix Checksum' button._ 

and a button "Fix Checksum".

**Comment by thenextman** [comment]
@memsharded It sounds like you are getting closer to this, but I'll add my feedback anyway. We don't use conan 2.0; we were previously on 1.43 but recently moved to 1.60. Our workflow is split into 2 (build and publish); due to an oversight we had build running on 1.60 but the publish was still running 1.43. However this worked at least once before hitting the problem, so I don't _think_ it's related. Now we have 1.60 everywhere.

**Comment by memsharded** [comment]
It seems the issue was fixed, and the patch has been (or is currently being) rolled out to the cloud instances, the repro has been ran on some affected instances and didn't reproduce anymore, so hopefully we are ok now.

Could you please confirm here when you verify the issue is gone? 
Thanks very much to all of you for your feedback, patience and support!

**Comment by SzBosch** [comment]
@memsharded: Which team and which patch were you taking about before? Conan or JFrog?
Can you please provide more details on the patch (incl. a link to a git repo when possible)?

We got an update of our Artifactory instance and it seems the problem is gone but we need some more time to verify.

What are the measures to avoid such problem will come up in future again?

**Comment by thenextman** [comment]
@SzBosch What version is your Artifactory at now? I noticed we got bumped to 7.67.4 and the revision incremented to 76704900.

**Comment by AldBosch** [comment]
@thenextman The version where we no longer see the issue is:
X-JFrog-Version: Artifactory/7.67.4 76704900

**Comment by thenextman** [comment]
@AldBosch Thank you, so we have the same patch version.

And @memsharded confirmed by jfrog with this message:

> With respect to the behavior you have encountered, this behavior is due to a known issue [...] With that said, you have been actively upgraded to our latest Artifactory patch, which includes the fix for this behavior.

**Comment by memsharded** [comment]
Yes, I meant the server side Artifactory 7.67.4 patch, that should solve the issue.

**Comment by fredizzimo** [comment]
Some new information, that I don't think I have seen here,
Our version of artifactory is 7.41.14 rev 7411490, and for me the files are not just zero bytes, they are missing completely. 

I'm also not sure why we haven't seen the problem until now.

**Comment by memsharded** [comment]
Hi @fredizzimo 

That seems a completely different issue, this was very specifically related to the latest 7.67 Artifactory release. I think we will close this issue soon as resolved (please everyone confirm), so if you are seeing that, please create a new ticket. Thanks!

**Comment by memsharded** [comment]
@drussel @thenextman @SzBosch can you please confirm if latest update solved the issues? Can we close this ticket?

**Comment by fredizzimo** [comment]
@memsharded, yes that's what I'm thinking as well now. The issue looks very similar, the debug log reports that the files are uploaded successfully, but they are not, and then I saw this issue thinking that it's the same. I'm still doing some local investigation, but if nothing turns up, I will write a new ticket.

**Comment by AldBosch** [comment]
@memsharded as me and @SzBosch work together, I can confirm that after the update we no longer see the issue.

**Comment by memsharded** [comment]
I am closing this ticket as solved, but please re-open or create a new ticket if this happens again.
Thanks all for your feedback!

**Comment by thenextman** [comment]
Confirmed fixed, thanks @memsharded
